hw-4/data-access/group.repository.ts

```diff
@@ -0,0 +1,79 @@
+import { Group, UserGroup } from '@app-models/index';
+import uuid from 'uuid';
+import sequelize from '../sequelize/sequalize';
+
+
+export default class GroupRepository {
```

Comment:
That's an awesome suggestion. Actually, I was scratching my head around this for a long time. I like approach you offer, but I still see concerns (or don't know how to implement it), because:
1. Models. At the moment they are tight coupled to sequelize, because 
    - it extends **Model** which is part of Sequelize package;
    - it calls .init() method and after definition and needs **sequelize** instance as a parameter.
2. All described above are required to make it possible to define relationships between those tables. In other words I need:
    - Create sequelize instance
    - Declare model A
    - Init model A
    - Declare model B
    - Init model B
    - Set relationship between A and B

It seems in this case I have to create POCO models (just plain properties I need) and use sequelize models as an implementation of those models for sequelize case... The more I think of it, the more questions I have actually. It becomes very complicated, no? WDYT?